### PR TITLE
Parsing Currency symbol in various places

### PR DESCRIPTION
**Pull request by kushthedude**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

-Currency symbol in add order and ticket summary was set to '$' by default.
-Changing the currency symbol as per the local currency.
#### Changes proposed in this pull request:

-Changed the standard '$' sign to local currency symbol.

Fixes #2225 , #2223

### CODE REVIEW

File: tests/integration/components/events/view/overview/event-tickets-test.js [hunk 1]
```diff
@@ -10,4 +10,10 @@ module('Integration | Component | events/view/overview/event tickets', function(
     await render(hbs`{{events/view/overview/event-tickets}}`);
     assert.ok(this.element.innerHTML.trim().includes('Tickets'));
   });
+  test('Currency Helper', async function(assert) {
+    this.set('paymentCurrency', '$');
+    await render(hbs`{{currency-symbol data.event.paymentCurrency}}`);
+    assert.equal(this.element.textContent.trim(), '$');
+  });
 });
```
Comment (kushthedude) [inline tests/integration/components/events/view/overview/event-tickets-test.js]: @CosmicCoder96  @iamareebjamal , A little help needed, Travis says not able read the 'property symbol' , Is something wrong with the test i added or i am completely on a wrong path.

Comment (kushthedude) [reply tests/integration/components/events/view/overview/event-tickets-test.js]: @shreyanshdwivedi If you could see some possible error in this , Why travis is failing?

**Comment by kushthedude** [comment]
@CosmicCoder96 There are many other places too where there is no currency parser or there is no data parser , Should i push commits of all such places in this PR or Should I open a new issue for every such place and then make a PR?

**Comment by CosmicCoder96** [comment]
@kushthedude Solve all the cases in this PR. Thanks!

**Comment by kushthedude** [comment]
> @kushthedude Solve all the cases in this PR. Thanks!

There are some places where the currency parser is not rendering and this is not letting other data values to be rendered , one of such place is given in #2402 ,
How should i tackle this @CosmicCoder96

File: tests/integration/components/events/view/overview/event-tickets-test.js (hunk 1 above)
Comment (kushthedude) [reply tests/integration/components/events/view/overview/event-tickets-test.js]: @uds5501 nope even after removing the test it tells unknown attribute ‘symbol’

**Comment by kushthedude** [comment]
@CosmicCoder96 i have noted all the places where the currency-symbol breaks the page, and where it works fine and there are some places where it dont break the page but the symbol does not gets renders too. I will make changes to those places where it is gettin rendered and will take a indepth look into it later where some workaround needs to be done

**Comment by CosmicCoder96** [comment]
@kushthedude can you please open an issue which lists the cases which remain after this PR is merged.

**Comment by kushthedude** [comment]
@CosmicCoder96 I will open the issue as soon as i get free from college😄

**Comment by kushthedude** [comment]
@CosmicCoder96 All the changes here are gettin rendered and not breakin the page

**Comment by CosmicCoder96** [comment]
but they still broke the build 🏗 alas

**Comment by kushthedude** [comment]
@CosmicCoder96 Currency symbol is not defined for event-ticket-test.js why dont we fix the test in #2444

**Comment by CosmicCoder96** [comment]
@kushthedude we don't merge PRs with failing builds. Remove the change which is failing the build. Add it to the list of places in #2444. That was the whole reason I asked you to open that issue. Just keep the changes which you can make without breaking anything including the build in this PR. I cannot spend more time on this.

**Comment by kushthedude** [comment]
@CosmicCoder96 done

**Pull request merged**